wire-ios-data-model/Source/Model/UserClient/UserClient.swift

```diff
@@ -421,6 +424,10 @@ public extension UserClient {
         client.activationDate = activationDate
         client.lastActiveDate = lastActiveDate
         client.remoteIdentifier = id
+        if let capabilities = payloadAsDictionary.optionalArray(forKey: "capabilities") as? [String],
```

Comment:
question: does this also need to be added to `UseClientLocalStore` when we update a client?

Reply:
I updated here https://github.com/wireapp/wire-ios/pull/2913/files#diff-f50c0403b3bff1afe8cce7f24ecc20342a242724f7cb2bdaae4d6f5a2223bb41R134, did you mean another place?